### PR TITLE
[WIP] Servo runner

### PR DESCRIPTION
**Pull request by victorporof**
Someone throw this away.

How to:
```sh
cd src/browser-runner/platform-servo
git clone git@github.com:servo/servo.git .servo
cd .servo
./mach build --dev
cd ../../../../
npm run servo+logging
```

screen shot 2017-04-25 at 6 11 25 pm

### CODE REVIEW

File: src/shared/util/spawn.js [hunk 1]
```diff
@@ -21,12 +21,25 @@ export const spawn = (command, main, args, { logger }, options = {}) => new Prom
     colors.blue(main),
     colors.yellow(args.join(' ')));
 
+  if (options.cwd) {
```
Comment (bgrins) [inline src/shared/util/spawn.js:24]: The changes in this file look useful - move into another PR?

Comment (victorporof) [reply src/shared/util/spawn.js:24]: Eh, ok.
